WordPress/src/main/java/org/wordpress/android/ui/accounts/login/nav/LoginNavHandler.java

```diff
@@ -0,0 +1,7 @@
+package org.wordpress.android.ui.accounts.login.nav;
+
+public interface LoginNavHandler {
+    void toast(String message);
```

Comment:
Just curious, is this a temporary method? If not, any reason we are not directly using the toast or `ToastUtils`?